aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -376,6 +376,65 @@ module aptos_token::token {
         *opt_in_flag = opt_in;
     }
 
+    public entry fun mutate_collection_description(
+        account: &signer,
+        creator: address,
+        collection_name: String,
+        description: String
+    ) acquires Collections {
+        assert!(signer::address_of(account) == creator, ENO_MUTATE_CAPABILITY);
+        assert!(exists<Collections>(creator), ECOLLECTIONS_NOT_PUBLISHED);
+        let collections = borrow_global_mut<Collections>(
+            creator
+        );
+        let collection_data = borrow_collection_data_mut(collections, collection_name);
+        // validate if the data is mutable
+        assert!(collection_data.mutability_config.description, EFIELD_NOT_MUTABLE);
```

Comment:
Wrap error code

Reply:
could you wrapper the code like this? it will provide much better error description when txn fail `        assert!(collection_data.mutability_config.description, error::permission_denied(EFIELD_NOT_MUTABLE));
`